dask/dataframe/io/parquet/core.py

```diff
@@ -347,7 +254,21 @@ def read_parquet_part(func, fs, meta, part, columns, index, kwargs):
     """ Read a part of a parquet dataset
 
     This function is used by `read_parquet`."""
-    df = func(fs, part, columns, index, **kwargs)
+    if isinstance(part, list):
+        concat_func = kwargs.pop("concat_func", False)
+        if not concat_func:
+            import pandas as pd
+
+            concat_func = pd.concat
+
+        dfs = []
+        for rg in part:
+            df = func(fs, rg, columns.copy(), index, **kwargs)
+            dfs.append(df)
```

Comment:
Can I ask for a list comprension here rather than a for loop?  

```python
if isinstance(part, list):
    dfs = [func(fs, rg, columns.copy(), index, **kwargs) for rf in part]
    df = methods.concat(dfs, axiis=0)
```